example/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.7",
+    "@react-native-community/netinfo": "^9.4.1",
```

Comment:
Since this is a native module can you add to the `readme` that they have to manually install this dependency?

In RN Native dependencies are not autolinked (only the NPM package is installed)